src/components/common/Breadcrumbs.js

```diff
@@ -0,0 +1,217 @@
+import {
+  Box,
+  IconButton,
+  Link,
+  MenuItem,
+  Popper,
+  Typography,
+} from '@mui/material';
+import React, { useMemo } from 'react';
+import Breadcrumbs from '@mui/material/Breadcrumbs';
+import { makeStyles } from '@mui/styles';
+import ArrowRightIcon from '@mui/icons-material/KeyboardArrowRight';
+import ArrowDownIcon from '@mui/icons-material/KeyboardArrowDown';
+import MoreHorizIcon from '@mui/icons-material/MoreHoriz';
+import vars from '../../assets/styles/variables';
+
+const { dropdownSecBg, breadcrumbLinkColor, breadcrumbTextColor } = vars;
+
+const useStyles = makeStyles(() => ({
+  crumb: {
+    fontWeight: 500,
+    fontSize: '0.875rem',
+    lineHeight: '1.25rem',
+    color: `${breadcrumbLinkColor} !important`,
+  },
+  expand: {
+    padding: '0.125rem',
+    borderRadius: '0.25rem',
+  },
+  lastCrumb: {
+    display: 'flex',
+    alignItems: 'center',
+    color: breadcrumbTextColor,
+
+    '& .MuiTypography-root': {
+      fontWeight: 500,
+      fontSize: '0.875rem',
+      lineHeight: '1.25rem',
+      color: 'inherit',
+    },
+  },
+  dropDown: {
+    width: '8rem',
+    maxWidth: 'content',
+    background: dropdownSecBg,
+    boxShadow:
+      '0 0.25rem 2.5rem rgba(0, 0, 0, 0.1), 0 1rem 7.5rem rgba(60, 60, 67, 0.2)',
+    padding: '0.375rem 0.25rem',
+    borderRadius: '0.5rem',
+    inset: '1rem auto auto 0 !important',
+
+    '&:before': {
+      content: '""',
+      width: 0,
+      height: 0,
+      borderStyle: 'solid',
+      borderWidth: '0 0.75rem 0.75rem 0.75rem',
+      borderColor: `transparent transparent ${dropdownSecBg} transparent`,
+      position: 'absolute',
+      top: '-0.5rem',
+      left: '3rem',
+    },
+
+    '& .MuiMenuItem-root': {
+      fontWeight: 500,
+      fontSize: '0.875rem',
+      lineHeight: '1.25rem',
+      whiteSpace: 'normal',
+      borderRadius: '0.375rem',
+      color: breadcrumbLinkColor,
+      padding: '0.25rem 0.5rem',
+
+      '&:hover': {
+        color: breadcrumbTextColor,
+      },
+    },
+  },
+}));
+
+const expand = 'expand';
```

Comment:
@emekauja thanks for this however constants like this should be in capitals and more explaining e.g. 
```
const EXPAND = 'expand';
```
in this case I would go for something like
```
const EXPAND_ID = 'expand';
```